### ISSUE TITLE
Problem when using Party().read('*')

### ISSUE THREAD

**Issue by imankahbasi** [eqcorrscan/EQcorrscan#387]
**Describe the bug**
I have parties in my directory.
When I use `p = Party().read('*')` to read all of them, `p` contains all families but not all detections.
It only have detections of last file.

**To Reproduce**
```python
>>> from eqcorrscan.core.match_filter.party import Party
>>> party = Party().read('party*tgz')

>>> print(party)
Party of 723 Families.
>>> print(party.get_catalog())
19 Event(s) in Catalog:

>>> import glob
>>> l = glob.glob('party*tgz')
>>> party = Party()
>>> for f in l:
>>>     party += Party().read(f)

>>> print(party)
Party of 723 Families.
>>> print(party.get_catalog())
461 Event(s) in Catalog
```
**Expected behavior**
It should have all detections when we use '*'.

**Desktop (please complete the following information):**
 - Operating System:
ubuntu 18.04.04
 - Python version:
3.7.3
 - EQcorrscan version:
0.4.0

**Comment by calum-chamberlain** [comment]
Read does not support wildcards and I'm surprised that anything is read in at all when you include the star.

If you want to implement wild card support you can,  but can you please do it for all the read functions.

CJ Chamberlain, out of office

________________________________

**Comment by imankahbasi** [comment]
> Read does not support wildcards and I'm surprised that anything is read in at all when you include the star.

But in tutorial described that `read` function in `Party()` supports wildcards.

![image](https://user-images.githubusercontent.com/47061763/80414009-9a8c5d80-88e5-11ea-872e-8dd45922c77f.png)


> If you want to implement wild card support you can,  but can you please do it for all the read functions.

Sure. But I don't have knowledge right now unfortunately. I will try to make it when I get enough skills.

**Comment by calum-chamberlain** [comment]
Ah, so I do, and so it should. Bugger!

Will test and fix at some point. Thanks for pointing this out.

**Comment by imankahbasi** [comment]
Your welcome.

**Comment by calum-chamberlain** [comment]
Closed by #453
